Packages/vcs/Lib/colormap.py

```diff
@@ -33,7 +33,7 @@ def process_src(nm, code):
     for i in range(255):
         val = numbers[i * 3:i * 3 + 3]
         if val != ():
-            d[i] = list(val)
+            d[i] = list(val)+[100]
```

Comment:
old colormaps only had r,g,b values so when loading back in it only stores 3 values, then later on when using getcolor we unpack it into r,g,b,a causing a failure. Voila.